ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -48,7 +49,8 @@ type ssh_daemon_options_type = {
     "ClientAliveInterval"               ? long
     "DenyGroups"                        ? string
     "DenyUsers"                         ? string
-    "HostKey"                           ? string
+    "GSSAPIStrictAcceptorCheck"         ? ssh_yesnostring
+    "HostKey"                           ? string[]
```

Comment:
We could have a separate schema for the multiline_options, instead of using the same one, I think this makes the most sense actually. This way we can leave HostKey as a string in options to keep compatibility, and put a string[] in multiline_options.

The Match blocks shouldn't be too difficult to support, hopefully we can just embed the existing structures inside another one.